libs/sdk/testing/src/lib/matchers/matchers.spec.ts

```diff
@@ -33,12 +33,20 @@ function createPassingElement(): any {
   return wrapper;
 }
 
-function createFailingElement(): any {
-  // Make every DIV have the same ID:
+function createFailingElement(params?: {
+  includeRelatedNodesError: boolean;
+}): HTMLDivElement {
+  // Use a deprecated role to trigger an accessibility failure.
   const element = createPassingElement();
-  [].slice.call(element.querySelectorAll('div')).forEach((elem: any) => {
-    elem.setAttribute('id', 'same-id');
-  });
+  element.setAttribute('role', 'directory');
+
+  if (params?.includeRelatedNodesError) {
+    // Related nodes include additional reporting.
+    const elem1 = element.firstElementChild as HTMLElement;
+    elem1.setAttribute('role', 'grid');
+    elem1.appendChild(document.createElement('textarea'));
+  }
+
```

Comment:
Changed b/c `duplicate-id` is no longer a failing error.

Reply:
Why not? 😅 Seems like it should still fail an accessibility check. Is our test not setup correctly to trigger the failure or did `axe-core` remove that as a rule?

Reply:
`axe-core` downgraded `duplicate-id` to `needs-review` rather than `failure`.

Reply:
It looks like they refined the error and created a new one, `duplicate-id-active` https://dequeuniversity.com/rules/axe/4.8/duplicate-id-active